### ISSUE TITLE
When I try use curl I get error 35

### ISSUE THREAD

**Issue by Andru1999** [rnixik/docker-openssl-gost#4]
I write comand `curl  https://portal.rosreestr.ru:4433 -k` and get error `curl: (35) error:14094410:SSL routines:ssl3_read_bytes:sslv3 alert handshake failure`. What I am doing wrong?

**Comment by rnixik** [comment]
Does this site work with any other software without a client certificate?

**Comment by Andru1999** [comment]
No. It shold work nly with authorization by certificate. And the user must also send his certificate too. I have not yet learned how to do it with curl, but I thought that I got other type of error.

**Comment by rnixik** [comment]
Could you try
    ```
    curl https://portal.rosreestr.ru:4433 -k -v --key key.pem --cert cert.pem
    ```
with your `key.pem` and  `cert.pem` and show output?

**Comment by Andru1999** [comment]
I have only cer file. No private key. How to get it from cer file?

**Comment by Andru1999** [comment]
No

**Comment by rnixik** [comment]
Do you have something like `----- ***** -----`?

**Comment by Andru1999** [comment]
No. I hava `0‚E0‚ф ч°zДЪ»з` in this file when I open it by notepad++

**Comment by rnixik** [comment]
It's probably in DER-format. You can try `openssl x509 -inform der -in certificate.cer -out certificate.pem` to convert it to pem and then look with notepad++.

**Comment by Andru1999** [comment]
Yes, it is works. I got `-----BEGIN CERTIFICATE-----`.

**Comment by rnixik** [comment]
Do you have `-----BEGIN PRIVATE KEY-----` somewhere in the file?

**Comment by Andru1999** [comment]
No. Mybe to need use special console key for get the private key ?

**Comment by rnixik** [comment]
Usually, obtaining private key is part of proccess of generating (obtaining) client certificate. Could you describe how did you get your cer?

**Comment by rnixik** [comment]
Currently, I don't know the reason of the error.

**Comment by Andru1999** [comment]
Maybe this servise use different algoritms? How to change connection algoritm?

**Comment by rnixik** [comment]
This server does not support TLS 1.1, TLS 1.2, TLS 1.3 but TLS1.0. May be it supports SSLv3, but curl in docker does not support it.
